modules/claims_api/app/sidekiq/claims_api/claim_establisher.rb

```diff
@@ -42,7 +42,7 @@ def perform(auto_claim_id) # rubocop:disable Metrics/MethodLength
 
       queue_flash_updater(auto_claim.flashes, auto_claim_id)
       queue_special_issues_updater(auto_claim.special_issues, auto_claim)
-    rescue ::Common::Exceptions::ServiceError => e
+    rescue ::Common::Exceptions::ServiceError, ::Common::Exceptions::BackendServiceException => e
```

Comment:
we won't really have the potential to rescue the `::Common::Exceptions::ServiceError` now, unless it comes from line :22 of `handle_error` now, so probably not necessary here.